src/Logary/Metric.fs

```diff
@@ -70,65 +75,61 @@ with
 
   override x.ToString() =
     sprintf "Metric(name=%s, type=%A, sampling=%O)"
-      x.name x.``type`` x.sampling
+      (PointName.format x.name) x.``type`` x.sampling
 
 /// Start configuring a metric with a metric factory
-let confMetric name sampling (factory : string -> Duration -> MetricConf) =
+let confMetric name sampling (factory : PointName -> Duration -> MetricConf) =
   factory name sampling
 
 let validate (conf : MetricConf) = conf
 
-/// initialises the configuration with the metadata to create a new metric-actor
+ /// initialises the configuration with the metadata to create a new metric-actor
 let init metadata conf =
   conf.initer metadata
 
 /// The GetValue implementation shall retrieve the value of one or more data
 /// points from the probe.
-let getValue (dps : DP list) =
-  Actor.reqReply
-    (fun chan -> GetValue(dps, chan))
-    Infinite
+let getValue (dps : PointName list) mi =
+  let resultCh = Ch ()
+  mi.requestCh *<+ GetValue (dps, resultCh) >>=.
+  resultCh
```

Comment:
The above has async client request paired with sync server reply, but without nack.  This is an unusual pattern for Hopac.

Typically one wants either

* async client request with nack (`reqCh *<+ (nack, repCh)`) and sync server reply (`repCh *<- result <|> nack`)

or

* sync client request (`reqCh *<- repIv`) and async server reply (`repIv *<= result`).

The first one has commit point on server reply and the second one has commit point on client request.  Both patterns are alternatives and can then be combined using `Alt` combinators.